demo/runtime/src/test_helper_pallet.rs

```diff
@@ -120,7 +133,25 @@ pub mod pallet {
 
 	impl<T: Config> pallet_partner_chains_bridge::TransferHandler<AccountId> for Pallet<T> {
 		fn handle_incoming_transfer(transfer: BridgeTransferV1<AccountId>) {
-			log::info!("Bridge transfer: {transfer:?}");
+			match transfer {
+				BridgeTransferV1::InvalidTransfer { token_amount, utxo_id } => {
+					log::warn!(
+						"⚠️ Recorded an invalid transfer of {token_amount} (utxo {utxo_id})"
+					);
+					TotalInvalidTransfers::<T>::mutate(|v| *v + token_amount);
+				},
+				BridgeTransferV1::UserTransfer { token_amount, recipient } => {
+					log::info!("💸 Registered a transfer of {token_amount} to {recipient:?}");
+					let _ = Balances::deposit_creating(&recipient, token_amount.into());
```

Comment:
Could you please tell me, if by querying storage of `balances` `accounts` will I see a trace of single token operation?

I'm thinking about testability.

Reply:
https://github.com/paritytech/polkadot-sdk/blob/978b35ebdcb5b02acce935a55a0dd5ef5798220e/substrate/frame/balances/src/impl_currency.rs#L426
As far as I see there is a standard Deposit event invoked which we can observe